src/components/NavDrawer.tsx

```diff
@@ -199,12 +205,15 @@ export const NavDrawer: React.FC<Props> = ({ children, logout }) => {
               component='div'
               key={"Logout"}
             >
-              <Tooltip title='Logout'>
+              <Tooltip title={`Logout (${profile?.username})`}>
```

Comment:
cc  @dkayiwa

Reply:
For the MVP, we are showing the username in the mouse over on Logout button. and in the expanded view, next to the Logout button. Screenshots are attached in the card.